src/Templates/default/html/directives/admonition.html.twig

```diff
@@ -1,5 +1,5 @@
 {# icons are from https://heroicons.com/ - MIT License #}
-<div class="admonition admonition-{{ name }} {{ class ?? '' }}">
+<div class="admonition admonition-{{ name }} {{ class }}">
```

Comment:
I dont think we need to check `null` here as we already set `null` be default in `AdmonitionDirective` - but i could totally be missing something obvious

https://github.com/weaverryan/docs-builder/blob/c40871d8664a23c2876777dcf42e4ae8d9bd421f/src/Directive/AdmonitionDirective.php#L27